js/index.js

```diff
@@ -28,6 +30,26 @@ function bindToForm(form) {
 	form.addEventListener('keyup', handleFormChanges)
 }
 
+const openFiscaRequestBuilder = (form) => () => {
```

Comment:
You here use `const xxx = () =>`, while using `function xxx()` below.

As stated earlier, I am in favour of named function declarations (`function xxx()`) and to keep usage of fat arrows for inline, unnamed functions. But, whichever you choose, be consistent.